kanidmd/src/lib/be/idl_sqlite.rs

```diff
@@ -2,6 +2,7 @@ use crate::audit::AuditScope;
 use crate::be::{BackendConfig, IdList, IdRawEntry, IdxKey, IdxSlope};
 use crate::entry::{Entry, EntryCommitted, EntrySealed};
 use crate::value::{IndexType, Value};
+use crate::{admin_error, admin_info, admin_warn, filter_error, spanned};
```

Comment:
You should consider adding these to prelude instead so you don't need to import them everywhere. :)